app/src/main/java/com/github/h3lp3rs/h3lp/AwaitHelpActivity.kt

```diff
@@ -265,4 +267,11 @@ class AwaitHelpActivity : AppCompatActivity() {
         // TODO should include deleting the conversation from the db
         startActivity(Intent(this, MainPageActivity::class.java))
     }
+
+
+    /** Starts the activity by sending intent */
+    private fun goToActivity(ActivityName: Class<*>?) {
+        val intent = Intent(this, ActivityName)
+        startActivity(intent)
+    }
 }
```

Comment:
I'm not sure to understand why you added this function, you don't seem to use it in this file.